File: py/exerc019.py
#**********************************************************************
# FILE NAME   : EXERC019                                              *
# DATE        : 2025-03-29                                            *
# AUTHOR      : FABIO MARQUES (FMARQUES@FMARQUES.ETI.BR)              *
# DATA CENTER : COMPANY.EDUC360                                       *
# PURPOSE     : SUM TWO 3x3 MATRICES AND DISPLAY RESULT               *
#**********************************************************************
import numpy as np

a    = np.zeros((3, 3))
b    = np.zeros((3, 3))
soma = 0

for i in range(3):
    for j in range(3):
        a[i, j] = float(input(f"Informe o valor para Matriz A[{i}][{j}]: "))

print ("***")

for i in range(3):
    for j in range(3):
        b[i, j] = float(input(f"Informe o valor para Matriz B[{i}][{j}]: "))

print ("***")

for i in range(3):
    for j in range(3):
        soma = a[i, j] + b[i, j]
        print (soma, "\t")

File: js/exerc019.js
/**********************************************************************
* FILE NAME   : EXERC018                                              *
* DATE        : 2025-03-29                                            *
* AUTHOR      : FABIO MARQUES (FMARQUES@FMARQUES.ETI.BR)              *
* DATA CENTER : COMPANY.EDUC360                                       *
* PURPOSE     : SUM TWO 3x3 MATRICES AND DISPLAY RESULT               *
**********************************************************************/

// Variable declaration
let a = Array(3)
a[0] = Array(3)
a[1] = Array(3)
a[2] = Array(3)

let b = Array(3)
b[0] = Array(3)
b[1] = Array(3)
b[2] = Array(3)

let c = Array(3)
c[0] = Array(3)
c[1] = Array(3)
c[2] = Array(3)

let soma       = 0
let maior_soma = 0
let linha      = 1

// Reading A matrix values
for (i = 0; i < 3; i++) {
  for (j = 0; j < 3; j++) {
    a[i][j] = parseFloat(prompt("Informe o valor para Matriz A[" + i + "][" + j + "]: "))
  } 
}

// Reading B matrix values
for (i = 0; i < 3; i++) {
  for (j = 0; j < 3; j++) {
    b[i][j] = parseFloat(prompt("Informe o valor para Matriz B[" + i + "][" + j + "]: "))
  } 
}
// Calulating the sum matrix
for (i = 0; i < 3; i++) {
  for (j = 0; j < 3; j++) {
    c[i][j] = a[i][j] + b[i][j]
  } 
}

// Display the sum matrix
console.log(c)